releasenotes/notes/1.2/port_star_prerouting-13fae3ff78feb5e3.yaml

```diff
@@ -1,11 +1,9 @@
 ---
 features_transpiler:
   - |
-    Port part of the logic from the :class:`StarPrerouting`, used to 
+    Port part of the logic from the :class:`.StarPreRouting`, used to 
     find a star graph connectivity subcircuit and replaces it with a 
     linear routing equivalent.
   - | 
-    The function :func:`star_preroute` now performs the heavily lifting
-    to transform the dag by in the rust space by taking advantage 
-    of the functions :func:`_build_sabre_dag` and 
-    :func:`_apply_sabre_result`.
+    The performance of :meth:`.StartPreRouting.star_preroute` is improved by performing the heavy 
+    lifting to transform the DAG in Rust space.
```

Comment:
This is a single release note right? Like we porting the logic of the pass to rust to speed it up. That's the release note here. I wouldn't call out the `star_preroute` method (tbh I'm not sure why that's public) and just leave this high level about the overall performance of the pass improving because of a porting to rust.